Ghidra/Processors/6805/data/languages/6x09.sinc

```diff
@@ -237,7 +237,8 @@ REL2: addr    is simm16 [ addr = inst_next + simm16; ]   { export *:2 addr; }
 # 1-byte operand, immediate/direct/indexed/extended addressing mode
 OP1: "#"imm8    is op45=0; imm8
 {
-    export imm8;
+    local tmp:1 = imm8;
+    export tmp;
```

Comment:
Rather than creating a new tmp variable, you should just do:
`export *[const]:1 imm8;`

Reply:
> Rather than creating a new tmp variable, you should just do: `export *[const]:1 imm8;`

Thanks, I didn't know that was possible.